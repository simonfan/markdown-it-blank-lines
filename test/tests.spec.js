import { readFileSync } from 'fs'
import { join } from 'path'

import markdownIt from 'markdown-it'

import blankLines from '../src'

const fixture = name => {
	return readFileSync(join(__dirname, 'fixtures', name), 'utf8')
}

describe('blank_lines', () => {
	test('snapshot-test', () => {
		const md = markdownIt().use(blankLines)
		const result = md.render(fixture('fixture-1.md'))

		expect(result).toMatchSnapshot()
	})
})
