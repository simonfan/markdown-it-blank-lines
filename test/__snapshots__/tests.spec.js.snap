// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`blank_lines snapshot-test 1`] = `
"<h1>Markdown</h1>
<p><br><br><br><br></p>
<p><br><br><br><br><br><br><br></p>
<p>Some text with blank_line token (~) ~in the middle~</p>
<p><br><br><br>Text in the middle of blank lines
<br></p>
<p>Last paragraph
<br></p>
<pre><code class=\\"language-javascript\\">~
~
~
~
</code></pre>
<blockquote>
<p>Blockquoted text</p>
<p><br><br><br></p>
<p>Blank lines within blockquote</p>
<p><br></p>
</blockquote>
"
`;
